metricbeat/module/php_fpm/_meta/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM richarvey/nginx-php-fpm:1.1.2
+FROM tsouza/nginx-php-fpm
```

Comment:
Can you also tag the image? So in case we change it in the future, it will not break older releases.